aip/general/0127.md

```diff
@@ -68,7 +68,7 @@ message CreateBookRequest {
   - RPCs **should** use the prescribed HTTP verb for custom methods, as
     discussed in [AIP-136][].
   - RPCs **should not** use `put` or `custom`.
-- The corresponding value represents the URI.
+- The corresponding value represents the path in the URI.
```

Comment:
(URN, in the nomenclature of https://en.wikipedia.org/wiki/Uniform_Resource_Identifier, though it seems this term is not commonly used)

Reply:
I suspect that far more readers will understand "path in the URI" than "URN".

Reply:
Agreed. (Hence the parentheses)